src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -488,9 +488,13 @@ private function getType(string $type, bool $isCollection, string $className = n
      */
     private function computeDoc(Documentation $documentation, \ArrayObject $definitions, \ArrayObject $paths) : array
     {
+        /** @var Symfony\Component\Routing\RequestContext $requestContext */
+        $requestContext = $this->urlGenerator->getContext();
```

Comment:
Hmm actually there's no need for any interface, because this is implementation-specific. So you can just inject the Symfony `RequestContext`.